src/App.js

```diff
@@ -1,26 +1,68 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
 
-function App() {
+import Board from './components/Board';
+
+const PLAYER_1 = 'x';
+const PLAYER_2 = 'o';
```

Comment:
Capital letters for extra intensity?

```suggestion
const PLAYER_1 = 'X';
const PLAYER_2 = 'O';
```